.github/workflows/routine_l10n_obsolete.yml

```diff
@@ -0,0 +1,56 @@
+name: Create l10n cleanup task
+on:
+  schedule:
+    - cron: 0 09 14 */3 *
```

Comment:
**praise (non-blocking):** Should this be `0 9 14 */2 *` ie, every other/second month, rather than every third month?

Reply:
Sure :)